### PR TITLE
Fix DF examples to make them work as notebooks

### PR DESCRIPTION
**Pull request by couet**
df107_SingleTopAnalysis.py:
  - fix arguments parsing
  - fix data file access

df106_HiggsToFourLeptons.py:
  -  fix data file access

Makefile:
  - Some DF tutorials need `PYSPARK_PYTHON` to be defined

### CODE REVIEW

File: tutorials/dataframe/df107_SingleTopAnalysis.py [hunk 1]
```diff
@@ -32,7 +33,12 @@
                     help="Use the full dataset (use --lumi-scale to run only on a fraction of it)")
 parser.add_argument("-b", action="store_true", default=False, help="Use ROOT batch mode")
 parser.add_argument("-t", action="store_true", default=False, help="Use implicit multi threading (for the full dataset only possible with --lumi-scale 1.0)")
-args = parser.parse_args()
+if 'df107_SingleTopAnalysis.py' in sys.argv[0]:
+    # Script
+    args = parser.parse_args()
+else:
+    # Notebook
+    args =parser.parse_args(args=[])
```
Comment (eguiraud) [inline tutorials/dataframe/df107_SingleTopAnalysis.py:41]:
```suggestion
    args = parser.parse_args(args=[])
```

**Review by eguiraud — APPROVED** [review]
LGTM

File: tutorials/dataframe/df107_SingleTopAnalysis.py (hunk 1 above)
Comment (couet) [reply tutorials/dataframe/df107_SingleTopAnalysis.py]: thanks

File: documentation/doxygen/Makefile [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 PYTHON_EXECUTABLE ?= /usr/bin/python3
 export PYTHON_EXECUTABLE
+export PYSPARK_PYTHON := $(PYTHON_EXECUTABLE)
```
[the remark this answers is not in the document]
Comment (couet) [reply documentation/doxygen/Makefile:6]: Yes, it is for one of the DF tutorials.

**Review by etejedor — APPROVED** [review]

**Pull request merged**